browser/help/topics/v4-hts.md

```diff
@@ -450,27 +450,27 @@ Row fields
-          - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
-          - `n_smaller`: Count of age values falling below lowest histogram bin edge.
-          - `n_larger`: Count of age values falling above highest histogram bin edge.
-- `exomes`: Struct of allele frequency information from the gnomAD exomes.
-  - `freq`: Array of allele frequency information (AC, AN, AF, homozygote count) for each frequency aggregation group in the gnomAD exomes.
+    - `age_hists`: Histograms containing age information for release samples.
+      - `age_hist_het`: Histogram for age in all heterozygous release samples calculated on high quality genotypes.
+        - `bin_edges`: Bin edges for the age histogram.
+        - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
+        - `n_smaller`: Count of age values falling below lowest histogram bin edge.
+        - `n_larger`: Count of age values falling above highest histogram bin edge.
+      - `age_hist_hom`: Histogram for age in all homozygous release samples calculated on high quality genotypes. If variant is in the pseudoautosomal regions of chrX or chrY, this histogram also includes age counts of hemizygous samples.
+        - `bin_edges`: Bin edges for the age histogram.
+        - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
+        - `n_smaller`: Count of age values falling below lowest histogram bin edge.
+        - `n_larger`: Count of age values falling above highest histogram bin edge.
+- `genomes`: Struct of allele frequency information from the gnomAD genomes.
+  - `freq`: Array of allele frequency information (AC, AN, AF, homozygote count) for each frequency aggregation group in the gnomAD genomes.
     - `AC`: Alternate allele count in release.
     - `AF`: Alternate allele frequency, (AC/AN), in release.
     - `AN`: Total number of alleles in release.
     - `homozygote_count`: Count of homozygous alternate individuals in release.
-  - `faf`: Filtering allele frequency in the gnomAD exomes.
+  - `faf`: Filtering allele frequency in the gnomAD genomes.
     - `faf95`: Filtering allele frequency (using Poisson 95% CI).
     - `faf99`: Filtering allele frequency (using Poisson 99% CI).
-  - `grpmax`: Allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry group with maximum allele frequency in the gnomAD exomes. Excludes Ashkenazi Jewish (`asj`), European Finnish (`fin`), Middle Eastern (`mid`), and "Remaining individuals" (`remaining`) groups.
+  - `grpmax`: Allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked genetic ancestry group with maximum allele frequency in the gnomAD genomes. Excludes Ashkenazi Jewish (`asj`), European Finnish (`fin`), Middle Eastern (`mid`), and "Remaining individuals" (`remaining`) groups.
```

Comment:
what about AMI for genomes?

Reply:
Oh, I didn't pay attention to the "AMI", let me double check. Then `mid` wasn't removed from exomes either, right? https://github.com/broadinstitute/gnomad_methods/blob/fde54e45017809c96c77a4da0381c28c3e41ad64/gnomad/resources/grch38/gnomad.py#L229

Reply:
ah, that's right -- this text would have been from v4.0 where we did remove mid